src/creatures/players/player.cpp

```diff
@@ -6567,8 +6577,44 @@ int64_t Player::getSkullTicks() const {
 	return skullTicks;
 }
 
-void Player::setSkullTicks(int64_t ticks) {
-	skullTicks = ticks;
+void Player::setSkullTicks(int64_t ticks) const {
+	const_cast<Player*>(this)->skullTicks = ticks;
```

Comment:
This needs to be reverted.
```suggestion
void Player::setSkullTicks(int64_t ticks) {
	skullTicks = ticks;
```